src/Avalonia.Animation/Animators/Animator`1.cs

```diff
@@ -33,7 +33,7 @@ public Animator()
         }
 
         /// <inheritdoc/>
-        public virtual IDisposable Apply(Animation animation, Animatable control, IClock clock, IObservable<bool> match, Action onComplete)
+        public virtual IDisposable? Apply(Animation animation, Animatable control, IClock? clock, IObservable<bool> match, Action? onComplete)
```

Comment:
Why does it return null?

Reply:
[This override](https://github.com/AvaloniaUI/Avalonia/blob/fixes/avalonia-animation-nullability/src/Avalonia.Visuals/Animation/Animators/TransformAnimator.cs#L91) can return null.

Reply:
tbh it should be a Disposable.Empty instead of just a null but im not sure if other places relies on it being null as well.